src/main/java/com/iota/iri/conf/NetworkConfig.java

```diff
@@ -9,68 +9,90 @@
 public interface NetworkConfig extends Config {
 
     /**
-     * @return Descriptions#NEIGHBORING_SOCKET_ADDRESS
+     * Default Value: {@value BaseIotaConfig.Defaults#NEIGHBORING_SOCKET_ADDRESS}
+     *
+     * @return {@value Descriptions#NEIGHBORING_SOCKET_ADDRESS}
      */
     String getNeighboringSocketAddress();
 
     /**
-     * @return Descriptions#NEIGHBORING_SOCKET_PORT
+     * Default Value: {@value BaseIotaConfig.Defaults#NEIGHBORING_SOCKET_PORT}
+     *
+     * @return {@value Descriptions#NEIGHBORING_SOCKET_PORT}
      */
     int getNeighboringSocketPort();
 
     /**
+     * Default Value: {@value BaseIotaConfig.Defaults#RECONNECT_ATTEMPT_INTERVAL_SECONDS}
      *
      * @return Descriptions#RECONNECT_ATTEMPT_INTERVAL_SECONDS
      */
     int getReconnectAttemptIntervalSeconds();
 
     /**
+     * Default Value: {@value BaseIotaConfig.Defaults#AUTO_TETHERING_ENABLED}
+     *
      * @return Descriptions#AUTO_TETHERING_ENABLED
      */
     boolean isAutoTetheringEnabled();
 
     /**
-     * @return Descriptions#P_REMOVE_REQUEST
+     * Default Value: {@value BaseIotaConfig.Defaults#P_REMOVE_REQUEST}
+     *
+     * @return {@value NetworkConfig.Descriptions#P_REMOVE_REQUEST}
      */
     double getpRemoveRequest();
 
     /**
-     * @return Descriptions#SEND_LIMIT
+     * Default Value: {@value BaseIotaConfig.Defaults#SEND_LIMIT}
+     *
+     * @return {@value NetworkConfig.Descriptions#SEND_LIMIT}
      */
     int getSendLimit();
 
     /**
-     * @return Descriptions#MAX_NEIGHBORS
+     * Default Value: {@value BaseIotaConfig.Defaults#MAX_NEIGHBORS}
+     *
+     * @return {@value Descriptions#MAX_NEIGHBORS}
```

Comment:
Normalize to `NetworkConfig.Descriptions`

Reply:
:+1: